Services/AuthentificationService/Controllers/AuthentificationController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AuthentificationService.Controllers
+{
+    [ApiController]
+    public class AuthentificationController : ControllerBase
+    {
+        /// <summary>
+        /// Storage for tokens 
+        /// May be used until database is created
+        /// </summary>
+
+        /// <summary>
+        /// Constructor for lulz and some data to play with
+        /// </summary>
+       /* public AuthentificationController()
```

Comment:
If we don't use it at all may be it's better to remove it rather than adding /**/